packages/ui/feature-builder-form-controls/src/lib/components/dropdown-search-control/dropdown-search-control.component.ts

```diff
@@ -0,0 +1,33 @@
+import { ChangeDetectionStrategy, Component, Input } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { UiCommonModule } from '@activepieces/ui/common';
+import { FormControl } from '@angular/forms';
+
+@Component({
+  selector: 'app-dropdown-search-control',
+  standalone: true,
+  imports: [CommonModule, UiCommonModule],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  template: `
+    <mat-form-field
+      subscriptSizing="dynamic"
```

Comment:
Remove

Reply:
remove what ? the subscriptSizing ? if I do that, it will add extra space under the control and make it look ugly.

Reply:
I mean just there is double subscriptSizingSizing